### ISSUE TITLE
[BUG] <radio volume does not adjust>

### ISSUE THREAD

**Issue by shannyshoez** [ParadoxGamingRP/bugs-and-suggestions#2506]
### Your Discord Name

shannyshoez

### A clear and concise description of what the bug is

when you raise the volume on your radio, it does not actually increase the volume to what you set it at. radio stays quiet

### Steps to Reproduce the Bug

1. louden radio volume
2. see that it does not set volume to what you put it to - doesn't update

### Expected Behavior

use the radio and set it to the volume you'd like it at

### Screenshots/Videos

n/a

**Comment by Darahz** [comment]
I just used mine and it works? Do you mean that its finicky?

**Comment by shannyshoez** [comment]
yes, ive had this issue since the wipe but only just reporting it bc i literally cant hear my ppl on radio LOL. if i crank the volume to all the way high, SOMETIMES it will bring the volume up a little bit. several others have told me they have the same issue

**Comment by shannyshoez** [comment]
so update: you can raise the volume with shift + right arrow. but the actually volume bar on the radios does not work properly
